manifest.webapp

```diff
@@ -65,10 +59,6 @@
       "type": "io.cozy.sharings",
       "verbs": ["ALL"]
     },
-    "triggers": {
-      "description": "Required to display additional information in the viewer and for the notification service",
-      "type": "io.cozy.triggers"
-    },
```

Comment:
I think this will break the capacity to execute services: you need to add the "POST" verb to the remaining triggers entry

Reply:
ne mettre aucun verbs c'est pas comme tous les avoir ?

Reply:
Si, je n'avais pas vu que avais enlevé le verbs pour les triggers. Ce serait bien de remettre `"verbs": ["GET", "POST"]` comme avant, vu que coachco2 n'a aucune raison d'avoir les droits de suppression sur les jobs/triggers

Reply:
ok je corrige

Reply:
Est-ce que CoachCO2 est considéré comme une app de plateforme ? Parce que là on retire des permissions (on passe de ALL à juste GET / POST) et donc la stack va bloquer l'update automatique.

Reply:
Ok pour avancer je supprime les verbs. C'est ce qu'on avait initialement https://github.com/cozy/coachCO2/pull/280/files#diff-a43db2645aeeb047b98024a5a2a9aa005d0cdbf439bdf1aa10c46a3d1cee9c42 avant qu'on fasse le duplicata en introduisant les verbs https://github.com/cozy/coachCO2/pull/277/files#diff-a43db2645aeeb047b98024a5a2a9aa005d0cdbf439bdf1aa10c46a3d1cee9c42

Reply:
Bah non, si Coach est considéré comme app de plateforme autant faire le changement. 

Et vu que ça l'est : https://github.com/cozy/cozy-stack/blob/afe48f05046eeca18f89d58b8dad759441fff3d9/pkg/consts/consts.go#L96C1-L108 

gardons les changements. 

Je voulais juste vous rappeler cette problématique, et que chercher l'info de savoir si app de plateforme ou pas est facile à trouver : soit stack soit vous demandez à l'infra la config.